index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
```

Comment:
- [ ] the title is not descriptive.

Reply:
- [ ] I think it would be better to mention the reason instead of saying 
   > the title is not descriptive
    you can say 
   >  I think the title is not descriptive because it needs to be descriptive and related to the project you are working on.